### ISSUE TITLE
Problem with 'accessibility_access true'

### ISSUE THREAD

**Issue by danielbayley** [caskroom/homebrew-cask#11699]
I added `accessibility_access true` to #11684 but now when I run `brew cask install finderminder` I get the following accessibility error;

```
==> Enabling accessibility access
==> Error: UNIQUE constraint failed: access.service, access.client, access.clien
Error: Command failed to execute!

==> Failed command:
["/usr/bin/sudo", "-E", "--", "/usr/bin/sqlite3", "#<Pathname:/Library/Application Support/com.apple.TCC/TCC.db>", "INSERT INTO access VALUES('kTCCServiceAccessibility','com.irradiatedsoftware.FinderMinder',0,1,1,NULL);"]
```

**Comment by vitorgalvao** [comment]
Can’t reproduce. Looks like an issue on your end.

Start by [following these](https://github.com/caskroom/homebrew-cask#reporting-bugs), and also see if you can reproduce with other casks with that option.

**Comment by danielbayley** [comment]
This is a strange one; other casks with `accessibility_access true` in them work fine, but not this one.

Following those instructions, here's the output of `brew cask install finderminder --verbose`
```bash
==> Downloading http://www.irradiatedsoftware.com/download/FinderMinder.zip
/usr/bin/curl -fLA Homebrew-cask v0.51+ (Ruby 2.0.0-481) http://www.irradiatedsoftware.com/download/FinderMinder.zip -C 0 -o /Library/Caches/Homebrew/finderminder-latest.zip.incomplete
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1153k  100 1153k    0     0   404k      0  0:00:02  0:00:02 --:--:--  404k
==> Symlinking App 'FinderMinder.app' to '/Applications/FinderMinder.app'
==> Enabling accessibility access
==> Error: UNIQUE constraint failed: access.service, access.client, access.client_type
Error: Command failed to execute!

==> Failed command:
["/usr/bin/sudo", "-E", "--", "/usr/bin/sqlite3", "#<Pathname:/Library/Application Support/com.apple.TCC/TCC.db>", "INSERT INTO access VALUES('kTCCServiceAccessibility','com.irradiatedsoftware.FinderMinder',0,1,1,NULL);"]

==> Output of failed command:


==> Exit status of failed command:
#<Process::Status: pid 8495 exit 19>
```

and
```
==> OS X Release:
10.10
==> OS X Release with Patchlevel:
10.10.3
==> Hardware Architecture:
intel-64
==> Ruby Version:
2.0.0-p481
==> Ruby Path:
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
==> Homebrew Version:
0.9.5
==> Homebrew Executable Path:
/usr/local/bin/brew
==> Homebrew Cellar Path:
/usr/local/Cellar
==> Homebrew Repository Path:
/usr/local
==> Homebrew Origin:
https://github.com/Homebrew/homebrew.git
==> Homebrew-cask Version:
0.54.0
==> Homebrew-cask Install Location:
/usr/local/Cellar/brew-cask/0.54.0
==> Homebrew-cask Staging Location:
/usr/local/opt/Caskroom
==> Homebrew-cask Cached Downloads:
/Library/Caches/Homebrew
/Library/Caches/Homebrew/Casks
2 files (warning: run "brew cask cleanup")
2.25 megs (warning: run "brew cask cleanup")
==> Homebrew-cask Default Tap Path:
/usr/local/Library/Taps/caskroom/homebrew-cask
==> Homebrew-cask Alternate Cask Taps:
/usr/local/Library/Taps/caskroom/homebrew-versions
==> Homebrew-cask Default Tap Cask Count:
2583
==> Contents of $LOAD_PATH:
/usr/local/Cellar/brew-cask/0.54.0/rubylib
/Library/Ruby/Site/2.0.0
/Library/Ruby/Site/2.0.0/x86_64-darwin14
/Library/Ruby/Site/2.0.0/universal-darwin14
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/x86_64-darwin14
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/universal-darwin14
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/x86_64-darwin14
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/universal-darwin14
==> Contents of $RUBYLIB Environment Variable:
<NONE>
==> Contents of $RUBYOPT Environment Variable:
<NONE>
==> Contents of $RUBYPATH Environment Variable:
<NONE>
==> Contents of $RBENV_VERSION Environment Variable:
<NONE>
==> Contents of $CHRUBY_VERSION Environment Variable:
<NONE>
==> Contents of $GEM_HOME Environment Variable:
<NONE>
==> Contents of $GEM_PATH Environment Variable:
<NONE>
==> Contents of $BUNDLE_PATH Environment Variable:
<NONE>
==> Contents of $PATH Environment Variable:
PATH="/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/Library/ENV/scm"
==> Contents of $SHELL Environment Variable:
SHELL="/bin/bash"
==> Contents of Locale Environment Variables:
LANG="en_GB.UTF-8"
==> Running As Privileged User:
No
```

**Comment by vitorgalvao** [comment]
Pinging @phinze @ndr-qef.

**Comment by ndr-qef** [comment]
I am guessing that `finderminder` had already been installed, or its accessibility already set. If so, SQLite cannot `INSERT` the record, as it already exist. The patch should be trivial.

**Comment by jawshooah** [comment]
Looks like this was already fixed in #11630, just need to push a new release.

**Comment by danielbayley** [comment]
Since I still have the local repo from adding formulas in `/usr/local/Library/Taps/caskroom/homebrew-cask` in Tower and have left it running, it periodically pulls from `origin/master`; so wouldn't #11630 be included anyway?

If so, i'm still having the same issue.

**Comment by vitorgalvao** [comment]
@danielbayley No. When @jawshooah means a new release, he is referring to [this](https://github.com/caskroom/homebrew-cask/releases), the core of homebrew-cask itself, the program that interacts with the individual casks.

**Comment by ndr-qef** [comment]
I'll cut a release later today.

**Comment by ndr-qef** [comment]
Fixed in v0.54.1. To upgrade, run `brew update && brew upgrade brew-cask`.
